### PR TITLE
Add scene view trainees

### PR DESCRIPTION
**Pull request by sgundlapalli11**

### CODE REVIEW

File: src/app/app-routing.module.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"use strict";
```
Comment (sgundlapalli11) [inline src/app/app-routing.module.js]: This any .js files will be removed

**Comment by hdang32** [comment]
@sgundlapalli11  You need to pull master to resolve conflicts

**Comment by sgundlapalli11** [comment]
> @sgundlapalli11 You need to pull master to resolve conflicts

Waiting for Jeannie's changes to be merged in first

File: src/app/components/scene-container/scene-container.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  selector: 'app-scene-container',
+  templateUrl: './scene-container.component.html',
+  styleUrls: ['./scene-container.component.scss']
+})
+export class SceneContainerComponent implements OnInit {
+  currentScene: Scene; // the input is the first scene to display
+  scenarios: Array<Scenario>;
+
+  updateScene(nextScene: number) {
+    console.log("The next scene is " + nextScene);
```
Comment (jeannie3) [inline src/app/components/scene-container/scene-container.component.ts]: This could be removed

**Pull request merged**